shared/django_apps/rollouts/models.py

```diff
@@ -1,17 +1,31 @@
-from random import choice
-
 from django.core.exceptions import ValidationError
 from django.db import models
 from django_better_admin_arrayfield.models.fields import ArrayField
 
 
-# TODO: move to utils
+# Defines whether flag is a front/back end feature flag. Backend
+# flags benefit from experimentation via `telemetry_simple` SQL metrics
+class Platform(models.TextChoices):
+    FRONTEND = "F", "Frontend"
+    BACKEND = "B", "Backend"
+
+
+# Defines the possible identifiers you can perform a rollout over. Users
+# who share the same identifier will always be assigned the same variant.
+# EG: two users from the same org will receive the same variant when rolling
+# out over ORG_ID
+class RolloutIdentifier(models.TextChoices):
+    OWNER_ID = "U", "Owner ID"
+    REPO_ID = "R", "Repo ID"
+    ORG_ID = "O", "Org ID"
+    EMAIL = "E", "Email"
```

Comment:
using "U" for OWNER_ID is a little awkward because we also have a `User` model which we could hypothetically also want to run an experiment over. a `User` is a user of codecov, and an `Owner` is an entity that can own repos on a git provider (so a user or org on GitHub, GitLab, or Bitbucket)

i think there will be few enough rows that we can use full words for these choices and it won't cost much space. also is there a benefit to having orgs and owners be separate?

Reply:
Yeah I can just make it the whole string then

Also, benefit of owners and orgs separate is just for clarity. I don't feel strongly about either decision to merge or keep them separate. Is it common knowledge that orgs are also owners?